### PR TITLE
COSS: Do not use .vol for volumes in which the pixel size is important to be stored in the header.

### PR DESCRIPTION
**Pull request by cossorzano**

### CODE REVIEW

**Comment by DStrelak** [comment]
Please follow the naming convention of the PR.
Also, please add reviewers

**Review by albertmena — APPROVED** [review]

**Comment by albertmena** [comment]
@cossorzano  Check my comments and we will close the PR or commit with changes.

File: xmipp3/protocols/protocol_multireference_alignability.py [hunk 1]
```diff
@@ -221,35 +232,34 @@ def convertInputStep(self, particlesId):
...
+        params +=  self.OUTPUTARG % self._getExtraPath('scaled_particles.stk')
         params +=  '  --save_metadata_stack %s' % self._getExtraPath('scaled_particles.xmd')
         params +=  '  --fourier %d' % newXdim
         
         self.runJob('xmipp_image_resize',params)
         
         from pwem.emlib.image import ImageHandler
         img = ImageHandler()
-        img.convert(self.inputVolumes.get(), self._getExtraPath("volume.vol"))
+        img.convert(self.inputVolumes.get(), self._getFileName("volume"))
```
[the remark this answers is not in the document]
Comment (cossorzano) [reply xmipp3/protocols/protocol_multireference_alignability.py:247]: I agree with you, but there is a problem with this: 1) Sonarcloud complains about code repetition, and 2) in Scipion it is promoted that, whenever possible, filenames are centralized in a single method and used as many times as needed along the protocol.

File: xmipp3/protocols/protocol_helical_parameters.py [hunk 2]
```diff
@@ -102,16 +110,16 @@ def coarseSearch(self):
         self.runCoarseSearch(self.fnVolSym,self.dihedral.get(),float(self.heightFraction.get()),
                              float(self.z0.get()),float(self.zF.get()),float(self.zStep.get()),
                              float(self.rot0.get()),float(self.rotF.get()),float(self.rotStep.get()),
-                             self.numberOfThreads.get(),self._getExtraPath('coarseParams.xmd'),
+                             self.numberOfThreads.get(),self._getFileName('coarse'),
```
[the remark this answers is not in the document]
Comment (cossorzano) [reply xmipp3/protocols/protocol_helical_parameters.py:113]: Same comment as for .vol

File: xmipp3/protocols/protocol_align_volume_and_particles.py [hunk 3]
```diff
@@ -159,7 +163,9 @@ def alignVolumeStep(self, maskArgs):
             args += " --frm"
         else:
             args += " --local"
-        args += " --copyGeo %s" % fhInputTranMat        
+        args += " --copyGeo %s" % fhInputTranMat
+        if not self.wrap:
```
[the remark this answers is not in the document]
Comment (cossorzano) [reply xmipp3/protocols/protocol_align_volume_and_particles.py:167]: It is in the form.

File: xmipp3/protocols/protocol_random_conical_tilt.py [hunk 4]
```diff
@@ -297,15 +298,25 @@ def createOutputStep(self):
         self._defineSourceRelation(self.inputParticlesTiltPair, self.volumesSet)
 
     def _appendOutputVolume(self, volumeOut):
+        fnMrc = volumeOut.replace(".vol",".mrc")
```
[the remark this answers is not in the document]
Comment (cossorzano) [reply xmipp3/protocols/protocol_random_conical_tilt.py]: Good idea. I have also moved convertToMrc to a common place (xmipp/convert.py)

**Review by albertmena — APPROVED** [review]

**Pull request merged**